docs/routing/dynamic-routes.md

```diff
@@ -67,15 +67,17 @@ Client-side navigations to a dynamic route can be handled with [`next/link`](/do
 
 Dynamic routes can be extended to catch all paths by adding three dots (`...`) inside the brackets. For example:
 
-- `pages/post/[...slug].js` matches `/post/a`, but also `post/a/b`, `post/a/b/c` and so on.
+- `pages/post/[...slug].js` matches `/post/a`, but also `/post/a/b`, `/post/a/b/c` and so on.
+
+> **Note**: `slug` is only a name, it can be anything, a catch all route is formed by having the brackets used by dynamic routes, and the three dots, and the page name (`[...param]`).
```

Comment:
```suggestion
> **Note**: `slug` is only a name, it can be anything, a catch all route is formed by having the brackets used by dynamic routes, the three dots, and a parameter name (`[...param]`).
```